webapp/components/Registration/CreateUserAccount.spec.js

```diff
@@ -56,6 +56,7 @@ describe('CreateUserAccount', () => {
             wrapper.find('input#name').setValue('John Doe')
             wrapper.find('input#password').setValue('hellopassword')
             wrapper.find('input#passwordConfirmation').setValue('hellopassword')
+            wrapper.find('input#checkbox').setChecked()
```

Comment:
Maybe you could add another test, expecting that if the terms and conditions checkbox is **not** checked the error message is displayed and the mutation is **not** called.

Reply:
Okay

Reply:
Okay II

Reply:
Okay III

Reply:
Okay IV

Reply:
Okay V

Reply:
Okay VI

Reply:
Okay VII